dev/io.openliberty.microprofile.reactive.messaging.internal_fat/fat/src/io/openliberty/microprofile/reactive/messaging/fat/kafka/metrics/MetricsTest.java

```diff
@@ -125,15 +125,14 @@ public static void teardown() throws Exception {
         }
     }
 
-    private HashMap<String, Integer> getMetrics() throws Exception {
```

Comment:
Why is this now static and why are you now passing in the server? The server is a static entity within the class?

Reply:
It's static so that we can reuse the method in the MultiApp test cases. Good spot on the server though, I had not noticed that